php/scripts/docker-entrypoint.sh

```diff
@@ -37,6 +37,11 @@ if [ "$BEHAT_SELENIUM_HOST" != "" ] && [ "$BEHAT_WEB_HOST" != "" ]; then
     fi
 fi
 
+## Disable xdebug if DISABLE_XDEBUG env is defined
+if [ -f ${PHP_INI_DIR}/conf.d/xdebug.ini ] && [ "${DISABLE_XDEBUG}" != "" ]; then
+    logger "Disabling xdebug"
+    rm -f ${PHP_INI_DIR}/conf.d/xdebug.ini
+fi
```

Comment:
I was kind of thinking opposite here, move xdebug into parent image, but disabled by default.

This way both images can enable debug, node image will still have debug disabled by default, we won't have to have separate dev intrim image, and afaik can get rid of `XDEBUG_CHANNEL` as we will be able to set it per PHP version.